### PR TITLE
add cloudflare workers template

### PR DESCRIPTION
**Pull request by zhelezkov**
Hey
Adding a template/example for cloudflare workers. Allows you to setup and deploy a your actions API pretty quickly
Demo: https://x.com/0xnze/status/1806343080758448218

### CODE REVIEW

File: examples/cloudflare-workers/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+if (globalThis.Buffer === undefined) {
+  globalThis.Buffer = Buffer;
+}
+
+// you should use a private RPC here
+const connection = new Connection("https://api.mainnet-beta.solana.com");
+
+const app = new Hono();
+
+app.use(cors());
```
Comment (nickfrosty) [inline examples/cloudflare-workers/src/index.ts:26]: Does this set the correct CORS headers required for actions and blinks to work? Including with the OPTIONS http method and the `actions.json` route?

File: examples/cloudflare-workers/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "solana-action-template",
+  "scripts": {
+    "dev": "wrangler dev",
+    "deploy": "wrangler deploy --minify",
+    "cf-typegen": "wrangler types --env-interface CloudflareBindings"
+  },
+  "dependencies": {
+    "@solana/actions": "^1.0.2",
```
Comment (nickfrosty) [inline examples/cloudflare-workers/package.json:9]: Can you update this the `1.0.3`? there was an typo in a constant that has been fixed

**Review by nickfrosty — APPROVED** [review]

File: examples/cloudflare-workers/src/index.ts (hunk 1 above)
Comment (zhelezkov) [reply examples/cloudflare-workers/src/index.ts:26]: it does!

**Comment by zhelezkov** [comment]
hey @nickfrosty, I've addressed all the comments. pls lmk if you need smth else from me to get it merged!

**Pull request merged**